### ISSUE TITLE
where is the `build.sh`

### ISSUE THREAD

**Issue by jander** [nim-lang/Nim#3738]

**Comment by jander** [comment]
the `install.txt` is old.

use `sh bootstrap.sh` to build nim on unix.

**Comment by dom96** [comment]
Those instructions apply to the .tar.xz file which can be downloaded from Nim's downloads page.

You're meant to [look at the Github readme](https://github.com/nim-lang/Nim/blob/devel/install.txt#L65) for instructions on how to build and install Nim from Github :)

**Comment by dom96** [comment]
That being said, perhaps that file should just be removed to avoid confusion.

**Comment by boydgreenfield** [comment]
Mmm shoot. I believe this needs to be updated in the Homebrew `nim.rb` formula, which I just submitted a PR for them to update...: https://github.com/Homebrew/homebrew/pull/48228

Sorry... I searched for "homebrew" in this repo but not nimble before updating and now see [this related issue](https://github.com/nim-lang/nimble/issues/173) in Nimble.

@dom96 preferences on fixing this all up vis-a-vis Homebrew + nimble support?
